From 97cfb4f1e751a947105802d5c4b4f56c696ff75d Mon Sep 17 00:00:00 2001
From: Michal Nowak <mnowak@startmail.com>
Date: Mon, 28 Oct 2019 15:38:34 +0100
Subject: Add cert bundle path for OpenIndiana

* lisp/net/gnutls.el (gnutls-trustfiles): Add cert bundle path for
OpenIndiana (bug#37960).

Copyright-paperwork-exempt: yes
---
 lisp/net/gnutls.el | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lisp/net/gnutls.el b/lisp/net/gnutls.el
index ae15b3a..9b13ada 100644
--- a/lisp/net/gnutls.el
+++ b/lisp/net/gnutls.el
@@ -111,6 +111,7 @@ Security'."
     "/usr/ssl/certs/ca-bundle.crt"           ; Cygwin
     "/usr/local/share/certs/ca-root-nss.crt" ; FreeBSD
     "/etc/ssl/cert.pem"                      ; macOS
+    "/etc/certs/ca-certificates.crt"         ; OpenIndiana
     )
   "List of CA bundle location filenames or a function returning said list.
 If a file path contains glob wildcards, they will be expanded.
-- 
cgit v1.0-41-gc330

